otwcode/otwarchive
# AO3-5250 when editing bookmark, submit button should say "update"

## Issue

https://otwarchive.atlassian.net/browse/AO3-5250

## Purpose

button now says `update` and not `edit` when saving changes to a bookmark while on a work or series

## Testing

1. Log in
2. Browse > Works
3. Access a work or series
4. Use the "Bookmark" link at the top of the work or series to open the bookmark form
5. Press "Create" to save your new bookmark
6. Access the work or series again
7. Use the "Edit Bookmark" link at the top of the work or series to open the bookmark form
8. Confirm the submit button says `Update` and not `Edit`
